### PR TITLE
Additional fixes after fact changes

### PR DESCRIPTION
**Pull request by drybjed**

### CODE REVIEW

**Comment by drybjed** [comment]
Not sure why Travis-CI doesn't report a successful test, but it should be fine. Merging.

File: ansible/roles/php/tasks/main_env.yml [hunk 1]
```diff
@@ -51,7 +51,8 @@
   set_fact:
     php__version: '{{ ansible_local.php.version|d(php__register_version.stdout) }}'
     php__long_version: '{{ ansible_local.php.long_version
-                           if (ansible_local.php.long_version|d() != "(none)")
+                           if (ansible_local|d() and ansible_local.php|d() and
+                               ansible_local.php.long_version|d() != "(none)")
```
Comment (ypid) [inline ansible/roles/php/tasks/main_env.yml:55]: Does the Ansible 2.8 shortcut not work here?

Comment (drybjed) [reply ansible/roles/php/tasks/main_env.yml:55]: Apparently, you can see the error [in the CI pipeline test](https://gitlab.com/debops/debops/-/jobs/601197359). I'm not sure what caused this, it also shows up with the `debug` module, so it's probably a quirk of Jinja.

Comment (ypid) [reply ansible/roles/php/tasks/main_env.yml:55]: Wired. Also does not make sense to me. Ok then.

**Pull request merged**